### DISCUSSION TITLE
Interpolating any value using ValueSchedule

### DISCUSSION THREAD

**Discussion by ssuukk** [FizzleDorf/ComfyUI_FizzNodes#88] [Q&A]
I tried to vary LoRA strength over frames using ValueSchedule, but it didn't seem to have the effect I expected. Is it possible to do this at all? And if yes - could you explain how?

**Comment by renwerx** [comment]
All the other LoRa schedulers I have found are for steps not frames. It seems like it would be pretty involved to implement this but combining LoRa travel with prompt travel would be one of the most powerful animation tools available if it worked.

**Comment by FizzleDorf** [comment]
@ssuukk you could try using the string schedules with the [prompt control](https://github.com/asagi4/comfyui-prompt-control) encoder nodes but the strength would be the only thing you could manipulate other than setting up a bunch of value schedules on conditioning average nodes. If all you need is something to do only the lora strengths, that would be the way to go.
